google/cloud/opentelemetry/internal/recordable.cc

```diff
@@ -184,9 +204,19 @@ void Recordable::SetResource(
     opentelemetry::sdk::resource::Resource const& /*resource*/) noexcept {}
 
 void Recordable::SetStartTime(
-    opentelemetry::common::SystemTimestamp /*start_time*/) noexcept {}
+    opentelemetry::common::SystemTimestamp start_time) noexcept {
+  // std::chrono::system_clock may not have nanosecond resolution on some
+  // platforms, so we avoid using it for conversions between OpenTelemetry
+  // time and Protobuf time.
+  auto t = absl::FromUnixNanos(start_time.time_since_epoch().count());
+  *span_.mutable_start_time() = internal::ToProtoTimestamp(std::move(t));
```

Comment:
`absl::Time` is one of those always-pass-by-value exceptions, so drop the `move()`.

Reply:
Done.